src/mame/drivers/zezinho.cpp

```diff
@@ -0,0 +1,47 @@
+// license:GPL-2.0+
+// copyright-holders:Felipe Sanches; Werner Moecke
+/*
+    Zezinho - Instituto Tecnológico da Aeronáutica (ITA)
+
+    Presumably the first computer designed and built in Brazil.
+    There are 2 variants. The first one, from 1961, is clearly not a turing-complete machine since
+    it only has got 4 instructions (add, subtract, data input and data output).
+
+    The second one, from 1962, was also called ITA-II, and has got 14 instructions including things like branching.
+    Unfortunately I am unaware of any software for the second iteration.
+
+    This driver only emulates the second variant, but runs a demo from the manual of the first one.
+    While there's now proper artwork, the only useful way to use this driver is by running it with MAME's debugger
+    to run the example code step-by-step.
```

Comment:
How this computer is supposed to run SW, tapes? Punch card? Other?
Feel free to link websites here if you have any.

Reply:
This was a very primitive computer. There's almost no info about it. It was probably programmed manually in some very rudimentary way of setting bits. We really lack more detailed docs.

Reply:
This is a wikipedia page about the computer, written in Brazilian Portuguese:
https://pt.wikipedia.org/wiki/Zezinho

Reply:
Hmm, this definitely needs some kind of planning, and a better picture resolution on reference [3] of the link attached for an artwork to happen. I also have concerns as UX standpoint around using a ROM, other than being "technically" wrong I mean.